docs/source/reference/links.rst

```diff
@@ -152,6 +152,44 @@ Classifier
 .. autoclass:: Classifier
    :members:
 
+Pre-trained models
+------------------
+
+Pre-trained models are mainly used to achieve a good performance with a small
+dataset, or extract a semantic feature vectors. Although ``CaffeFunction``
+automatically loads a pre-trained model released as a caffemodel,
+the following link models provide an interface for automatically converting
+caffemodels, and easily extracting semantic feature vectors.
+
+For example, to extract the feature vectors with ``VGG16Layers``, which is
+a common pre-trained model in the field of image recognition,
+users need to write the following few lines::
+
+    from chainer.links import VGG16Layers
+    from PIL import Image
+
+    model = VGG16Layers()
+    img = Image.open("path/to/image.jpg")
+    feature = model.extract([img], layers=["fc7"])["fc7"]
+
+where ``fc7`` denotes a layer before the last fully-connected layer.
+Unlike the usual links, these pre-trained models automatically load all the
+parameters from the pre-trained models during initialization.
```

Comment:
No. This class loads the parameters from both pre-trained caffe models (``.caffemodel``) and pre-trained chainer models ``.npz`` depending on the situation.